pisa/core/map.py

```diff
@@ -939,9 +939,14 @@ def fluctuate(self, method, random_state=None, jumpahead=0):
         Parameters
         ----------
         method : None or string
-            Valid strings are '', 'none', 'poisson', 'gauss', or
+            Valid strings are '', 'none', 'poisson', 'scaled_poisson', 'gauss', or
             'gauss+poisson'. Strings are case-insensitive and whitespace is
             removed.
+            The 'scaled_poisson' method implements a Scaled Poisson Process, which is 
+            a better approximation to the true distribution when the bin counts are 
```

Comment:
Better approximation compared to what? Which "true" distribution? You mean the true distribution of the random variates due to a finite number of MC events, right? I keep coming back to this because pisa originally wasn't intended to be able to reweight MC events. This shouldn't hold up the PR, but we should work on documentation (also, and in particular, for the other methods) at some point.